### DISCUSSION TITLE
There is no VAT number . Its GST going forward. This block needs to be changed in the checkout page

### DISCUSSION THREAD

**Discussion by tintsandtoons** [woocommerce/woocommerce#53456] [Checkout Flow]
There is no VAT number . Its GST going forward. This block needs to be changed in the checkout page

**Comment by nielslange** [comment]
Hello @tintsandtoons 👋,

Thank you for reaching out! By default, the Checkout block doesn’t include a field for a VAT number. However, in [this GitHub discussion](https://github.com/woocommerce/woocommerce/discussions/51177#discussioncomment-11276189), I shared an example of how to add a custom field for the VAT number to the Checkout block.

You also mentioned, “It's GST going forward.” Can you clarify what you mean by that? Are you referring to the Goods and Services Tax? Would you require both VAT and GST fields?

If so, you can extend the code example I shared to add both fields. If you only need one, you can simply customise the field's label and name it VAT or GST as required.

I hope this helps! If you have any additional questions, feel free to reach out.
